package.json

```
@@ -9,7 +9,7 @@
     "build": "tsc",
     "format": "eslint --fix \"src/**/*.ts\"",
     "generate-vib-client": "openapi-generator-cli generate -c openapitools.json --generator-key vib",
-    "lint": "eslint \"src/**/*.ts\"",
+    "lint": "eslint --resolve-plugins-relative-to \"src/**/*.ts\"",
```

I think `--resolve-plugins-relative-to` is breaking the linter.

I checked it in the following scenario:
```
$ git diff
diff --git a/src/action.ts b/src/action.ts
index d61da81..f31367b 100644
--- a/src/action.ts
+++ b/src/action.ts
@@ -15,6 +15,8 @@ import AdmZip from "adm-zip"
 import { Readable } from "stream"
 import { randomUUID } from "crypto"
 
+let variable;
+
 export interface ActionResult {
   baseDir: string,
   artifacts: string[],
```
The following detects the lint error:
```
$ eslint src/**/*.ts
...
  1 error and 0 warnings potentially fixable with the `--fix` option.
```
But this does not:
```
$ eslint --resolve-plugins-relative-to  src/**/*.ts
...
$
```

---

`"lint": "eslint --resolve-plugins-relative-to \"src/**/*.ts\" \"src/**/*.ts\""` seems to work though

---

this seems to work though; not sure if that would help as I do not know why it is necessary to define a path for plugins
```suggestion
    "lint": "eslint --resolve-plugins-relative-to \"src/**/*.ts\" \"src/**/*.ts\"",
```